nilearn/plotting/data/html/report_head_template.html

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
any particular reason for using XHTML? other nilearn plots are in html5, maybe we should be consistent. btw, at the moment the generated reports are not valid XHTML

Reply:
I can drop the XHTML here or add it in to the other plots ! Personal preference would be to drop here for now and have another PR to sync all plots. WDYT @GaelVaroquaux ?